docs/content/how-to/kubevirt/create-kubevirt-cluster.md

```diff
@@ -53,31 +86,59 @@ export PULL_SECRET="$HOME/pull-secret"
 export MEM="6Gi"
 export CPU="2"
 export WORKER_COUNT="2"
+export BASE_DOMAIN=hypershift.lab
+# OCP_RELEASE is optional, this will determine which version is used for the Hosted Cluster
+# if not set, it will default to the hardcoded version in the hypershift cli
+export OCP_RELEASE=4.12.7-x86_64
 
 hypershift create cluster kubevirt \
 --name $CLUSTER_NAME \
---node-pool-replicas=$WORKER_COUNT \
+--node-pool-replicas $WORKER_COUNT \
 --pull-secret $PULL_SECRET \
 --memory $MEM \
---cores $CPU
+--cores $CPU \
+#--release-image quay.io/openshift-release-dev/ocp-release:$OCP_RELEASE \
+--base-domain $BASE_DOMAIN
 ```
 
 !!! note
 
     A default NodePool will be created for the cluster with 2 vm worker replicas
     per the `--node-pool-replicas` flag.
 
+After a few moments we should see our hosted control plane pods up and running:
+
+~~~sh
+oc -n clusters-$CLUSTER_NAME get pods
+
+NAME                                                  READY   STATUS    RESTARTS   AGE
+capi-provider-5cc7b74f47-n5gkr                        1/1     Running   0          3m
+catalog-operator-5f799567b7-fd6jw                     2/2     Running   0          69s
+certified-operators-catalog-784b9899f9-mrp6p          1/1     Running   0          66s
+cluster-api-6bbc867966-l4dwl                          1/1     Running   0          66s
+.
+.
+.
+redhat-operators-catalog-9d5fd4d44-z8qqk              1/1     Running   0          66s
+~~~
+
 A guest cluster backed by KubeVirt virtual machines typically takes around 10-15
 minutes to fully provision. The status of the guest cluster can be seen by
 viewing the corresponding HostedCluster resource. For example, the output below
 reflects what a fully provisioned HostedCluster object looks like.
 
+In order for the hosted cluster to complete its deployment, we need to fix the Ingress, we will do that in the next sections.
+
```

Comment:
the default settings are meant to avoid people needing to consider ingress

Reply:
Okay, will reorganize the document so the default example doesn't require ingress.